pennylane/pauli/dla/structure_constants.py

```diff
@@ -116,6 +119,28 @@ def structure_constants(
     >>> adjoint_rep[:, 0, 1] # commutator of X_0 and Y_0 consists of first and last operator
     array([-2.,  0.,  2.])
 
+    We can also use matrix representations for the computation, which is sometimes faster, in particular for sums of many Pauli words.
+    This is just affecting how the structure constants are computed internally, it does not change the result.
+
+    >>> adjoint_rep2 = qml.structure_constants(dla, is_orthogonal=False, matrix=True)
+    >>> qml.math.allclose(adjoint_rep, adjoint_rep2)
+    True
+
+    We can also input the DLA in form of matrices. For that we use :func:`~lie_closure` with the ``matrix=True``.
```

Comment:
Is this suggestion correct?
```suggestion
    We can also input the DLA as a list matrices. For that we use :func:`~lie_closure` with ``matrix=True``.
```